frontend/packages/noobaa-storage-plugin/src/components/capacity-card/capacity-card-body.tsx

```diff
@@ -65,21 +65,26 @@ export const CapacityCardBody: React.FC<CapacityCardBodyProps> = ({ isLoading, m
   if (isLoading) {
     return <LoadingInline />;
   }
+
   // trying to avoid redundant code
   const { metricData, metricLegend, total } = getMetricDataAndLegend(metricsData);
   if (!metricData.length) {
     return (
-      <EmptyState className="chart-empty-state" variant={EmptyStateVariant.full}>
-        <EmptyStateIcon size="sm" icon={ChartPieIcon} />
-        <Title size="sm">No Prometheus datapoints found</Title>
-      </EmptyState>
+      <div className="noobaa-capacity-card__item">
+        <div className="co-capacity-card__item-title">Storage</div>
```

Comment:
this is not right. `GaugeChart` is using PF4 `ChartUtilizationDonut`, but you want PF4 `ChartDonut`.
Also title in error/loading state is `Storage` and in ok state its `Usage Breakdown`.

Title should be same and I think in this case you still want to use ChartDonut but change title to `Loading` or `No Data`